doc/develop/languages/c/common_libc.rst

```diff
@@ -1,4 +1,4 @@
-.. _c_library_common
+.. _c_library_common:
```

Comment:
This should have been part of the `doc/develop: Document common C library APIs`.

Reply:
thanks; I missed this in an initial push and amended the wrong patch.